tests/e2e/setup-contracts.mjs

```diff
@@ -18,31 +27,42 @@ const merkleRootToMint = {
   tokenRegistry: [
     {
       // Endorse Owner
-      merkleRoot: "0xf82cf3c92462175888eda124a5227e0a46344ba09650e4ef85015d97e342251a",
+      merkleRoot: "0x5759d368034cd5a316178344b66eb511f10edf4bd5d633dd1c8862a647fc6f4b",
       ...defaultToken,
     },
     {
       // Nominate Owner
-      merkleRoot: "0x764fb454298834edfdb2d169f8397d7a6e4c4b997f872e6a0808e4a2e2bdb00e",
       ...defaultToken,
+      merkleRoot: "0xfc4271d1fe0219bc76b693f9e9f9808264800c54c69502988491455499c15188",
       holder: ADDRESS_EXAMPLE_2,
     },
     {
       // Surrender
-      merkleRoot: "0xfb4a84f4b9e8e28c552eab1d94f5d4a831584082478f9fa83a2e8aac61e07c4a",
+      merkleRoot: "0xb969686a6bbd635f02b7c0073ac4a787ee54f93c2feaf94ea600dc0b798aa574",
       ...defaultToken,
     },
     {
       // Transfer Holder
-      merkleRoot: "0x1c92119b530b5028e8cb45e74624a5f52a1ae84d68e1c165776813a84722807a",
+      merkleRoot: "0x3a7436eac11bb9a98f75de7743e70e99b4f1a2625b372449b83979cf8e9b5183",
       ...defaultToken,
     },
   ],
 };
 
+shell.exec(`${oaCLI_PATH} deploy document-store "My Document Store" -n local -k ${ACCOUNT_KEY}`);
```

Comment:
in synpress flow, we are only testing token registry / ebl fixtures, no need for deploy doc store

Reply:
yup, will clean up.